scripts/core_build.sh

```diff
@@ -101,6 +101,7 @@ USE_ASAN="OFF"
 USE_DYNAMIC_SIMD="ON"
 USE_OPENDAL="OFF"
 INDEX_ENGINE="KNOWHERE"
+: "${ENABLE_GCP_NATIVE:="ON"}"
```

Comment:
Since there is another dependency involved into Milvus. Let's set by default OFF

Reply:
Done. Please check.